ewlegal.cls

```diff
@@ -0,0 +1,84 @@
+%% TODO Stock beginning text, copyrights and project info and all that
+
+%% Work consists of the files engwales-legal.cls and accompanying engwales-*.sty
+
+
+\NeedsTeXFormat{LaTeX2e}
+\ProvidesClass{ewlegal}[2021-02-21 Legal Templates for England and Wales]
+
+%% Uses article as a base
+\LoadClass[12pt,a4paper]{article}% TODO Perhaps use KOMA-Script?
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Options
+%%%%%
+
+%% Easy option management
+\RequirePackage{kvoptions}
+
+%% Select the template used
+% TODO Be able to nicely handle multiple templates being selected
+
+\DeclareBoolOption[false]{pleading}
+
+\DeclareBoolOption[false]{opinion}
+
+
+%% Extras
+
+
+
+%% Process options given by the user
+\ProcessKeyvalOptions*
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Packages
+%%%%%
+
+%% Generic Packages
+
+%% Easy formatting of section titles
+\RequirePackage[explicit]{titlesec}
+
+%% Allows for easy use of logic
+\RequirePackage{etoolbox}
+
+%% Fancy headers and footers
+\RequirePackage{fancyhdr}
+
+%% Spaced out boxes
+\RequirePackage{soul}
+% Doesn't seem to be reccommended anymore due to issues with luatex
+
+% What does this do?
+\RequirePackage[normalem]{ulem}
+
+%% Expand the feature set of typeset lists
+\RequirePackage{enumitem}
+
+%% Handle margins
+\RequirePackage[margin=2.5cm]{geometry}
+% Should the end user always have to declare this? Or should we try for
+% reasonable defaults?
+
+
+%% Engwales-legal Packages
+
+\RequirePackage{ewlegal-utils}
+
+\ifbool{ewlegal@pleading}{\RequirePackage{ewlegal-plead}}{}
+
+\ifbool{ewlegal@opinion}{\RequirePackage{ewlegal-opinion}}{}
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Typesetting
+%%%%%
+
+%% Set linespacing to be 1.5 % Seems to be the most commonly suggested
+\linespread{1.5}
```

Comment:
note that this is different than 1.5 spacing in Word tho. this is closer to double line space in Word. @passamo9 is Word 1.5 line spacing required?

Reply:
opps, that's a typo, meant to be 1.3.

Reply:
i find `\onehalfspacing` more appealing but again that's different than Word's 1.5 spacing, so i'm not sure if it's allowed.

Reply:
From the [link](https://www.civillitigationbrief.com/2017/11/05/fonts-lawyers-and-the-rules-never-ever-use-comic-sans/) @passamo9 sent in #29, it doesn't look like there is any rigid regulation, but merely preferred practice. So, I doubt that there'd be any penalty due to the word vs latex 1.5 line spacing difference.